packages/design-system/src/styles/components/_UsaBanner.scss

```diff
@@ -1,189 +1,208 @@
+      // Additional padding needed to accommodate close button in expanded state
+      // Button is only visible in mobile/tablet viewports
+      padding-inline-end: 3.5rem;
     }
   }
 
-  .ds-c-usa-banner__cta-wrapper {
-    display: block;
+  @media (min-width: $media-width-md) {
+    align-items: center;
+    // Changing display:grid to flex for larger viewports
+    // The button goes from being a full-width element on mobile/tablet to something that looks like it's an inline link with text and needs to flow appropriately. Flex accomplishes this.
+    display: flex;
+    min-height: auto;
   }
 }
 
-.ds-c-usa-banner__collapse-banner-container {
-  align-items: center;
-  background-color: var(--usa-banner-close__background-color);
-  display: flex;
-  height: $spacer-6;
-  justify-content: center;
-  margin-block-start: calc(-1 * #{$spacer-1});
-  margin-inline: auto calc(-1 * #{$spacer-2});
-  width: $spacer-6;
-
-  .ds-c-icon {
-    height: var(--font-size-lg);
-    width: var(--font-size-lg);
-  }
+.ds-c-usa-banner__header-icon {
+  grid-area: icon;
+  height: 1em;
+  width: 1em;
 }
 
-.ds-c-usa-banner__header-flag {
-  height: 11px;
-  margin-inline-end: $spacer-1;
-  width: $spacer-2;
+.ds-c-usa-banner__header-text {
+  grid-area: text;
+  margin: 0;
 }
 
-.ds-c-usa-banner__header-text {
-  display: inline;
+.ds-c-usa-banner__action {
+  color: var(--usa-banner-link__color);
+  grid-area: action;
   margin: 0;
-  span {
-    margin-inline-end: $spacer-1;
+  text-decoration: underline;
+
+  .ds-c-usa-banner__header--expanded & {
+    // When banner is expanded on mobile/tablet, action text goes away
+    display: none;
   }
 }
 
-.ds-c-usa-banner__header-action {
-  color: var(--usa-banner-action__color);
-  display: block;
-  margin-block: 2px 0;
-  text-decoration: underline;
+.ds-c-usa-banner__action-icon {
+  color: currentColor;
+  height: 0.75em;
+  margin-inline-start: 0.5ch;
+  // Needed to overwrite the vertical alignment of the icon so the action text appears aligned with the banner text. Applied utility flex classes on the action text to re-implement vertical alignment.
+  vertical-align: initial;
+  width: 0.75em;
 }
 
 .ds-c-usa-banner__button {
   background-color: transparent;
   border: 0;
-  bottom: 0;
-  box-shadow: none;
-  margin: 0;
-  padding: 0;
-  text-align: start;
+  color: var(--usa-banner-action__color);
+  cursor: pointer;
+  inset: 0;
+  inset-inline-start: initial;
+  position: absolute;
+  // Mobile/tablet, button is full-size of banner
+  width: 100%;
 
   &:focus {
     @include mixins.focus-styles;
   }
+
+  @media (min-width: $media-width-md) {
+    display: inline-block;
+    height: fit-content;
+    line-height: 1.1;
+    padding: 0;
+    position: relative;
+    text-align: start;
+    width: fit-content;
+  }
 }
 
 .ds-c-usa-banner__button-text {
-  @extend .ds-c-link;
+  // This element is screen-reader only on mobile/tablet
+  @extend .ds-u-visibility--screen-reader;
```

Comment:
I tried putting this in a media query (`@media (max-width: $media-width-md) {}`) but discovered you can't `@extend` in a `@media` - o how the limitations of SCSS vex me so!

Reply:
Hmm, maybe it's better to use the utility class in this case. It would be easier to debug in the future. I know we want to limit use of utility classes in our own component code, but I feel like the screen reader utility is an exception

Reply:
Now that I've seen your other comments about overriding this later, I understand why you did this

Reply:
Should we actually make a mixin for the screen reader styles so we can use them exactly where and how we want? Would we then be able to put them inside a media query with max width like you suggested?

Reply:
A mixin is worth exploring... looking into that now!